### ISSUE TITLE
s3cmd info shows ETag as MD5 sum

### ISSUE THREAD

**Issue by hohoqq** [s3tools/s3cmd#1336]
s3cmd info shows ETag as MD5 sum. But "The entity tag may or may not be an MD5 digest of the object data." https://docs.aws.amazon.com/AmazonS3/latest/API/API_CompleteMultipartUpload.html#API_CompleteMultipartUpload_ResponseSyntax

So the MD5 sum in the output of s3cmd may not be the MD5 sum of the file. It's misleading.

![image](https://github.com/s3tools/s3cmd/assets/64099513/bc95c530-c1b0-4dfc-9632-6320735e7ff9)

**Comment by lavigne958** [comment]
yes AWS S3 documentation must be open about the value stored in the ETag.

So far on regular object it is an opaque value but in fact it is the md5 sum of the object data. 

They need to stay open as some upload will store some different values. example: when putting an object and requesting AWS S3 to encrypt your object, the ETag is NOT the md5 sum of the object data. for this kind of scenarios they must stay open in the actual value used here in the documentation.

I still agree with you S3cmd should rename that display to `ETag` to be consistent.
